src/weights.jl

```diff
@@ -27,6 +27,12 @@ isempty(wv::AbstractWeights) = isempty(wv.values)
 Base.getindex(wv::AbstractWeights, i) = getindex(wv.values, i)
 Base.size(wv::AbstractWeights) = size(wv.values)
 
+function Base.isequal(w₁::W, w₂::W) where {W <: AbstractWeights}
```

Comment:
Most of the existing `isequal` methods I've seen use `x` & `y` or `a` & `b` instead of  subscripts.